### PR TITLE
"Ancestor" Announcer voice pack

### PR DESCRIPTION
**Pull request by Snuke01**
## About The Pull Request
Adds The Ancestor (voiced by Wayne June) as a possible announcer. uses voicelines from Darkest Dungeon.

## Why It's Good For The Game
Honestly it probably isnt. but I made this while I was drunk, and now I realise its too late for me NOT to try this PR. I guess people might get a kick out of having a new announcer. Also Wayne June's voice is fucking sexy.

## Changelog
:cl:

soundadd: added The Ancestor trait voice pack

/:cl:

### CODE REVIEW

**Comment by amidoingitright** [comment]
ruin has come to Austation

**Comment by Kapu1178** [comment]
can we steal this for upstream

**Comment by Snuke01** [comment]
> 
> 
> can we steal this for upstream

go for it :)

File: code/datums/station_traits/neutral_traits.dm [hunk 1]
```diff
@@ -71,10 +72,25 @@
...
 
 /datum/station_trait/announcement_baystation/New()
 	. = ..()
 	SSstation.announcer = /datum/centcom_announcer/baystation
+
+
+/datum/station_trait/announcement_ancestor
+	name = "Announcer: Ancestor"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 1
```
Comment (Archanial) [inline code/datums/station_traits/neutral_traits.dm:87]: This makes it as rare if not rarer than joke intern announcer. I'd suggest bumping it to 2 or 3.

Comment (Snuke01) [reply code/datums/station_traits/neutral_traits.dm:89]: i wanted the trait to be incredibly rare, so people wouldnt get bored of it so soon.

**Review by MCterra10 — APPROVED** [review]

**Comment by MCterra10** [comment]
actually wait this isn't modular

File: beestation.dme [hunk 2]
```diff
@@ -416,6 +416,7 @@
 #include "code\datums\announcers\default_announcer.dm"
 #include "code\datums\announcers\intern_announcer.dm"
 #include "code\datums\announcers\medbot_announcer.dm"
+#include "code\datums\announcers\ancestor_announcer.dm"
```
Comment (MCterra10) [inline beestation.dme:419]: this should be added in austation.dme

File: code/datums/announcers/ancestor_announcer.dm [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+						ANNOUNCER_METEORS = 'sound/ai/ancestor/meteors.ogg',
+						ANNOUNCER_OUTBREAK5 = 'sound/ai/ancestor/outbreak5.ogg',
+						ANNOUNCER_OUTBREAK7 = 'sound/ai/ancestor/outbreak7.ogg',
+						ANNOUNCER_POWEROFF = 'sound/ai/ancestor/poweroff.ogg',
+						ANNOUNCER_POWERON = 'sound/ai/ancestor/poweron.ogg',
+						ANNOUNCER_RADIATION = 'sound/ai/ancestor/radiation.ogg',
+						ANNOUNCER_SHUTTLECALLED = 'sound/ai/ancestor/shuttlecall.ogg',
+						ANNOUNCER_SHUTTLEDOCK = 'sound/ai/ancestor/shuttledock.ogg',
+						ANNOUNCER_SHUTTLERECALLED = 'sound/ai/ancestor/shuttlerecalled.ogg',
+						ANNOUNCER_SPANOMALIES = 'sound/ai/ancestor/spanomalies.ogg')
```
Comment (MCterra10) [inline code/datums/announcers/ancestor_announcer.dm:21]: please move this file to the modular austation folder

File: code/datums/station_traits/neutral_traits.dm [hunk 4]
```diff
@@ -71,10 +73,26 @@
...
+	show_in_report = TRUE
+	report_message = "Wayne June has generously donated his time to provide some spooky announcement lines for the season"
+	blacklist = list(/datum/station_trait/announcement_medbot,
+	/datum/station_trait/announcement_baystation, /datum/station_trait/announcement_intern
+	)
+
+/datum/station_trait/announcement_ancestor/New()
+	. = ..()
+	SSstation.announcer = /datum/centcom_announcer/ancestor
```
Comment (MCterra10) [inline code/datums/station_traits/neutral_traits.dm:98]: please put this in a new modular austation file

File: code/datums/station_traits/neutral_traits.dm [hunk 5]
```diff
@@ -43,7 +43,8 @@
 	show_in_report = TRUE
 	report_message = "Please be nice to him."
 	blacklist = list(/datum/station_trait/announcement_medbot,
-	/datum/station_trait/announcement_baystation
+	/datum/station_trait/announcement_baystation,
+	/datum/station_trait/announcement_ancestor
```
Comment (MCterra10) [inline code/datums/station_traits/neutral_traits.dm:47]: please either identify the changes with a comment as specified in the readme.md file in the austation folder:
OR
override the variables in a modular austation code file
### Comment Formatting
```dm
// austation -- <change reason>
```
for single line changes
and
```dm
// austation start -- <change reason>
<code>
// austation end
```
for multiline

File: code/datums/station_traits/neutral_traits.dm [hunk 6]
```diff
@@ -58,7 +59,8 @@
 	report_message = "Our announcement system is under scheduled maintanance at the moment. Thankfully, we have a backup."
 	blacklist = list(
 		/datum/station_trait/announcement_intern,
-		/datum/station_trait/announcement_baystation
+		/datum/station_trait/announcement_baystation,
+		/datum/station_trait/announcement_ancestor
```
Comment (MCterra10) [inline code/datums/station_traits/neutral_traits.dm:63]: as above

File: code/datums/station_traits/neutral_traits.dm [hunk 7]
```diff
@@ -71,10 +73,26 @@
 	weight = 5
 	show_in_report = TRUE
 	report_message = "We lost the primary datatape that holds the announcement system's voice responses. We did however find an older backup."
-	blacklist = list(/datum/station_trait/announcement_intern,
+	blacklist = list(/datum/station_trait/announcement_intern, 
+	/datum/station_trait/announcement_ancestor,
```
Comment (MCterra10) [inline code/datums/station_traits/neutral_traits.dm:77]: as above

**Review by MCterra10 — CHANGES_REQUESTED** [review]
all the sound files should be moved to `austation/sound` too

**Pull request closed without merging**